src/app/pages/OnDemandTvPage/VideoPlayer/index.jsx

```diff
@@ -19,19 +15,8 @@ import { ServiceContext } from '#contexts/ServiceContext';
 import getEmbedUrl from '#lib/utilities/getEmbedUrl';
 import getPlaceholderImageUrl from '../../../routes/utils/getPlaceholderImageUrl';
 
-const VideoPlayerWrapper = styled.div`
-  width: calc(100% + ${GEL_SPACING_DBL});
-  margin: 0 -${GEL_SPACING};
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    width: calc(100% + ${GEL_SPACING_QUAD});
-    margin: 0 -${GEL_SPACING_DBL};
-  }
-`;
-
-const landscapeRatio = '56.25%';
-const MediaMessageWrapper = styled.div`
-  padding-top: ${landscapeRatio};
-  position: relative;
```

Comment:
Are you sure we don't need these styles any more?  Unless something has changed in <MediaMessage> it will look fairly strange now